recipes/raylib/all/conanfile.py

```diff
@@ -17,13 +18,19 @@ class RaylibConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "build_examples": [True, False],
+        "use_external_glfw": ["OFF","ON","IF_POSSIBLE"],
```

Comment:
we may use boolean `True`/`False` instead of "ON"/"OFF" string. "IF_POSSIBLE" I wouldn't add, it's something not reproducible as I understand